### DISCUSSION TITLE
Code coverage report

### DISCUSSION THREAD

**Discussion by Sumanharapanahalli** [bxparks/AUnit#122] [Q&A]
Hi,

How can I get a code coverage report with this when testing on targets?

Best Regards,
Suman Harapanahalli

**Comment by bxparks** [comment]
AUnit does not support code coverage by itself.

You *might* be able to get code coverage if you run AUnit using EpoxyDuino on the desktop. I've never tried it though.

You *might* be able to use AUnit under PlatformIO. Here's an article I found with my first google search: https://piolabs.com/blog/insights/test-coverage-on-unit-testing.html. Again, never tried it.

Code coverage is not a subject that I am particularly interested in, so I cannot add much info. You will have to do your own research
